drivers/scheduler/k8s/k8s.go

```diff
@@ -692,6 +777,19 @@ func (k *K8s) createCoreObject(spec interface{}, ns *v1.Namespace, app *spec.App
 		return dep, nil
 
 	} else if obj, ok := spec.(*apps_api.StatefulSet); ok {
+		// Add security annotations if running with auth-enabled
+		if configMapName != "" {
+			configMap, err := k8sOps.GetConfigMap(configMapName, "default")
```

Comment:
will the config map always be in default namespace?

Reply:
Yes, currently we create the config map in the jenkins job in "default" namespace. 
Should we have the namespace as parameter?